crawler/build.sbt

```diff
@@ -0,0 +1,43 @@
+name := "newspaper-crawler"
+
+version := "1.0"
+
+scalaVersion := "2.12.1"
```

Comment:
Yes, but being able to jump to definitions is very handy when doing code review, so I needed to switch to unstable ENSIME 2.x. Well, one step closer to contributing to that project ;)

In the worst case I can easily work without IDE-like features, I've grown up without them.

So from my perspective it doesn't really matter much. Just pointing out that if we're going to use `2.12`, then we can use `2.12.2` already, no need for `2.12.1`. I'm staying with `2.11.11` for now.